### PR TITLE
docs: update Readme and Contributing docs

### LINKED ISSUE #401: Getting started docs
We should update the readme with commands for getting started.

### PR DESCRIPTION
**Pull request by kohidave**
Update the Readme docs with installation instructions as well
as a getting started example.

Take a look here: https://github.com/kohidave/amazon-ecs-cli-v2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Resolves #401

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,72 @@
...
+  --deploy
+```
+
+This will create a VPC, Application Load Balancer, an Amazon ECS Service with the sample app running on AWS Fargate. This process will take around 8 minutes to complete - at which point you'll get a URL for your sample app running!
+
+<details>
+    <summary> watch what happens when you run <tt>init</tt></summary>
+
+![Step By Step Setup](https://user-images.githubusercontent.com/828419/69770895-91813f80-113f-11ea-8be9-60df6c2bf3fc.gif)
+</details>
```
Comment (efekarakus) [inline README.md]: :O cool! I had no idea this was a thing!

File: README.md [hunk 2]
```diff
@@ -1,27 +1,72 @@
...
+
+### Cleaning up 🧹
+
+Once you're finished playing around with this project, you can delete it and all the AWS resources associated it by running `ecs-preview project delete`.
+
+### Learning more 📖
+
+Want to learn more about what's happening? Check out our [getting started guide](https://github.com/aws/amazon-ecs-cli-v2/wiki/Getting-Started) and a detailed breakdown of all the [infrastructure](https://github.com/aws/amazon-ecs-cli-v2/wiki/Infrastructure) that gets created.
+
+## Tenets 🌟
```
Comment (efekarakus) [inline README.md]: Should we move this to a `CHARTER.md` like firecracker does? https://github.com/firecracker-microvm/firecracker/blob/master/CHARTER.md

File: README.md [hunk 3]
```diff
@@ -1,27 +1,72 @@
...
 
 The ECS CLI is a tool for developers to create, release and manage production ready containerized applications on ECS.
-From getting started, pushing to staging and releasing to production, the ECS CLI can help manage the entire lifecycle
-of your application development.
+From getting started, pushing to a test environment and releasing to production, the ECS CLI helps you through the entire life of your app development.
 
-Once you've built something you're excited to deploy, let the ECS CLI set up a CI/CD pipeline for you,
-with built in testing hooks and manual gates.
-Tail your logs, monitor your key metrics and push updates all from the comfort of your terminal.
+Got a Dockerfile and some code? Get it up and running on ECS in under 10 minutes, with just one command. Ready to take that app to production? Spin up new environments and a continuous delivery pipeline without having to leave your terminal. Find a bug? Tail your logs and deploy with one tool.
```
Comment (efekarakus) [inline README.md]: I like that one sentence summary above, do you think we should have this text under:
`## What's the ECS CLI?"` similar to https://github.com/firecracker-microvm/firecracker

File: README.md [hunk 4]
```diff
@@ -35,9 +80,15 @@ The CLI must support workflows around troubleshooting and debugging to help when
 * **Deliver applications continuously.**
 While the ECS CLI can be used to manually deploy changes to an application, we always help customers to move to CI/CD by helping them set up and manage pipelines.
 
+## We need your feedback 🙏
+
+The ECS CLI v2 is in developer preview, meaning we want to know what works, what doesn't work, and what you want! Have any feedback at all? Drop us an [issue](https://github.com/aws/amazon-ecs-cli-v2/issues/new) or join us on [gitter](https://gitter.im/aws/amazon-ecs-cli-v2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
+
+We're happy to hear feedback or answer questions, so reach out, anytime!
+
 #### Security disclosures
```
Comment (efekarakus) [inline README.md]: should this be a h2? for consistency

**Comment by efekarakus** [comment]
oh can you add "Resolves: https://github.com/aws/amazon-ecs-cli-v2/issues/401" in the commit

File: README.md (hunk 2 above)
Comment (kohidave) [reply README.md]: The Tenets section? I can do that :D

**Pull request merged**